Xamarin.Essentials/AppInfo/AppInfo.ios.tvos.watchos.cs

```diff
@@ -23,5 +23,17 @@ static string GetBundleValue(string key)
         static void PlatformShowSettingsUI() =>
             throw new FeatureNotSupportedException();
 #endif
+
+        static WindowSize PlatformWindowAppSize()
+        {
+#if __IOS__ || __TVOS__
+            var currentView = Platform.GetCurrentViewController().View.Window?.Frame;
+
+            if (currentView == null)
+                return default;
+
+            return new WindowSize(currentView.Value.Width, currentView.Value.Height);
```

Comment:
I'm probably being dense here, so feel free to ignore without comment, but I'm not sure how this will compile on platforms other than IOS or TVOS ... I'd have thought you'd need a #else throw FeatureNotSupportedException

Reply:
I close this PR because I did something wrong, please review it [here](https://github.com/xamarin/Essentials/pull/993)

And tks for your review :)